.gitmodules

```diff
@@ -2,8 +2,10 @@
   path = spack
   #url = https://github.com/spack/spack
   #branch = develop
-  url = https://github.com/jcsda/spack
-  branch = jcsda_emc_spack_stack
+  #url = https://github.com/jcsda/spack
+  #branch = jcsda_emc_spack_stack
+  url = https://github.com/ashley314/spack
+  branch = feature/add_sphinx
```

Comment:
@ashley314 the change here is temporary and is done to help CI testing find your PRs feature branch. I named the feature branch in this PR the same as yours but that isn't necessary.

The idea here is:
- Once testing passes here, merge your PR in spack
- Then revert these changes (in .gitmodules)
- Then update the submodule commit hash to the one after your PR was merged
- Then merge this PR (once tests pass and reviewers approve of course)

Note that if you are only making changes in spack-stack (without changes in spack) then you don't need to go through this process because in this case the spack submodule commit hash remains the same.

Reply:
I should save this writeup because it's so short and clear. Thanks, Steve!

Reply:
Yes awesome explanation! Thanks for clarifying :)